queue_job/jobrunner/channels.py

```diff
@@ -474,7 +474,7 @@ def set_done(self, job):
         This removes it from the channel queue.
         """
         self.remove(job)
-        _logger.debug("job %s marked done in channel %s", job.uuid, self)
+        _logger.info("job %s marked done in channel %s", job.uuid, self)
```

Comment:
Are you sure about this change, it doesn't match your commit description :)

> stop jobrunner channels' log flooding

Reply:
Indeed, there's a commit that should not be there